src/components/AuthPage/Login/index.js

```diff
@@ -1,28 +1,46 @@
-import React, { useEffect, useState } from "react";
-import { useFirebase } from "react-redux-firebase";
-import { Link } from "react-router-dom";
-import { clearAuthError, signIn } from "../../../store/actions";
 import {
-  Form,
-  Input,
-  Button,
-  Typography,
-  Row,
-  Col,
   Checkbox,
-  Divider,
-} from "antd";
-import { MailOutlined, LockOutlined } from "@ant-design/icons";
-import SmButtons from "../smButtons";
-import { useSelector, useDispatch } from "react-redux";
-import ViewAlerts from "./ViewAlerts";
-const { Title } = Typography;
+  FormControlLabel,
+  FormGroup,
+  Grid,
+  IconButton,
+  InputAdornment,
+  TextField,
+  Typography,
+} from '@material-ui/core';
```

Comment:
Hi @hiranthaR,

Can you please follow this link https://github.com/scorelab/Codelabz/pull/75#discussion_r638543643 and do the necessary changes.

Follow [this comment](https://github.com/scorelab/Codelabz/pull/75#discussion_r638703932) also.

Reply:
Sure. I'll change them. and thanks for the information :heart: